src/gui/optionsdialog.cpp

```diff
@@ -299,17 +299,18 @@ void OptionsDialog::loadBehaviorTabOptions()
     m_ui->checkPreventFromSuspendWhenDownloading->setChecked(pref->preventFromSuspendWhenDownloading());
     m_ui->checkPreventFromSuspendWhenSeeding->setChecked(pref->preventFromSuspendWhenSeeding());
 
-    m_ui->checkFileLog->setChecked(app()->isFileLoggerEnabled());
+    const bool fileLogEnabled = app()->isFileLoggerEnabled();
+    m_ui->checkFileLog->setChecked(fileLogEnabled);
     m_ui->textFileLogPath->setDialogCaption(tr("Choose a save directory"));
     m_ui->textFileLogPath->setMode(FileSystemPathEdit::Mode::DirectorySave);
     m_ui->textFileLogPath->setSelectedPath(app()->fileLoggerPath());
     const bool fileLogBackup = app()->isFileLoggerBackup();
     m_ui->checkFileLogBackup->setChecked(fileLogBackup);
-    m_ui->spinFileLogSize->setEnabled(fileLogBackup);
+    m_ui->spinFileLogSize->setEnabled(fileLogEnabled && fileLogBackup);
```

Comment:
Problem was that groubox disabled + `m_ui->checkFileLogBackup` enabled had this spinbox enabled.